src/dependency_groups/__main__.py

```diff
@@ -1,4 +1,3 @@
-#!/usr/bin/env python
```

Comment:
These files are not executable, so having a shabang doesn't help. I could have made them executable and left the shabang in if you'd prefer.